src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -63,10 +64,11 @@ bool ShouldEnableReflectionPatternReporting (MethodDefinition method)
 			return !_context.Annotations.HasLinkerAttribute<RequiresUnreferencedCodeAttribute> (method);
 		}
 
-		public ReflectionMethodBodyScanner (LinkContext context, MarkStep parent)
+		public ReflectionMethodBodyScanner (LinkContext context, MarkStep parent, MarkScopeStack scopeStack)
```

Comment:
nit: should be `in`

Reply:
`MarkScopeStack` is a `class` - does `in` do anything interesting on a ref type?

Reply:
My bad for some reason I though it's a struct